tests/e2e/support.py

```diff
@@ -348,3 +349,47 @@ def get_nodes_by_label(self, node_labels):
     label_selector = ",".join(f"{k}={v}" for k, v in node_labels.items())
     nodes = self.api_instance.list_node(label_selector=label_selector)
     return [node.metadata.name for node in nodes.items]
+
+
+def assert_get_cluster_and_jobsubmit(
+    self, cluster_name, accelerator=None, number_of_gpus=None
+):
+    # Retrieve the cluster
+    cluster = get_cluster(cluster_name, self.namespace)
+
+    cluster.details()
+
+    cluster.config.verify_tls = False
```

Comment:
This is unnecessary as you can specify `verify_tls` in the `get_cluster` function